lib/add_dsi_users_to_spreadsheet.rb

```diff
@@ -0,0 +1,65 @@
+require 'dfe_sign_in_api'
+require 'spreadsheet_writer'
+
+class AddDSIUsersToSpreadsheet
+  def initialize
+    @worksheet = Spreadsheet::Writer.new(DSI_USER_SPREADSHEET_ID, DSI_USER_WORKSHEET_GID, true)
```

Comment:
I prefer inverting dependencies.

Reply:
@C-gyorfi @Renny4Real A little note for you to take a look at :)